src/modules/app/StoreActionType.js

```diff
@@ -52,6 +53,15 @@ export type PushCategoryActionType = {|
     path: string, depth: number, key: string
   |}
 |}
+export type PushCategoryLanguagesActionType = {|
+  type: 'PUSH_CATEGORY_LANGUAGES', params: {|
```

Comment:
I think this name is a bit unclear as it also sets the status 'languagesNotAvailable'.

Maybe a name like 'CATEGORY_LANGAUGE_NOT_AVAILABLE'. Or something more generic 'SET_CATEGORY_STATUS'.

Reply:
Yeah, I'm also not quite happy with my naming here... 
But `CATEGORY_LANGUAGE_NOT_AVAILABLE` is not an action and `SET_CATEGORY_STATUS` is too generic. I might go with the first one though...

Reply:
Would it be possible to just use the `FetchCategoryFailedAction` with an additional param `languageNotAvailable` and set the route status accordingly? Language not  available leads to a failed fetch :man_shrugging:

Reply:
I used the FetchCategoryFailedAction (and corresponding for events) now.